tools/isobuild/meteor-npm.js

```diff
@@ -644,22 +644,30 @@ var updateExistingNpmDirectory = function (packageName, newPackageNpmDir,
   var shrinkwrappedDependenciesTree =
     getShrinkwrappedDependenciesTree(packageNpmDir);
 
-  const npmTree = { dependencies: {} };
+  let npmTree = { dependencies: {} };
   _.each(npmDependencies, (version, name) => {
     npmTree.dependencies[name] = { version };
   });
 
-  const minInstalledTree =
+  let minInstalledTree =
     minimizeDependencyTree(installedDependenciesTree);
-  const minShrinkwrapTree =
+  let minShrinkwrapTree =
     minimizeDependencyTree(shrinkwrappedDependenciesTree);
 
+  const npmTreeHadFileReference = hasFileReference(npmTree);
+  minShrinkwrapTree = removeFileDependencies(minShrinkwrapTree, npmTree);
+  npmTree = removeFileDependencies(npmTree, npmTree);
+
   if (isSubtreeOf(npmTree, minInstalledTree) &&
       isSubtreeOf(minShrinkwrapTree, minInstalledTree)) {
     return;
   }
 
-  if (! quiet) {
+  if (!quiet) {
+    if (!_.isEmpty(minInstalledTree.dependencies)) {
+      console.log("\nnpmTree", JSON.stringify(npmTree, null, 4), "minInstalledTree", JSON.stringify(minInstalledTree, null, 4),
```

Comment:
This produces a rather cryptic verbose output when running `./startdebug.sh`. Could we say something more meaningful here? So that user running the script could understand what is going on.